src/components/HomePage/Banner.js

```diff
@@ -68,8 +68,10 @@ const useStyles = makeStyles(theme => ({
 		justifyContent: 'center',
 		alignItems: 'baseline',
 		alignSelf: 'flex-end',
+		padding: '25px 0px 0px 0px',
```

Comment:
For spacing, we like to use the `theme.spacing()` function so we get consistent units and if we ever need to scale up or down the spacing we'll be able to do it across all the files in `layout.js` by overriding the default style.

The default spacing in material UI is 8px, so let's turn this into the closest factor of 8 with `theme.spacing()`, which would be 24px. (3 * 8 = 24px)
```suggestion
		padding: theme.spacing(3, 0, 0, 0),
```